jakarta/integrationtest/java/modules/orm-coordination-outbox-polling-elasticsearch/pom.xml

```diff
@@ -46,6 +46,12 @@
             <artifactId>hibernate-search-mapper-orm-coordination-outbox-polling-jakarta</artifactId>
         </dependency>
 
+        <!-- Only needed here so we can require it in module info for Avro -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
```

Comment:
Same comment as elsewhere, shouldn't this be `log4j-api`, and can we reduce the scope?